src/Gallery-get.js

```diff
@@ -4,6 +4,8 @@ import deathNote from './lightdeathnoteposter.jpg';
 import gameOfThrones from './GameOfThrones.jpg';
 import walkingDead from './The-Walking-Dead-Season-3.jpg';
 import wire from './TheWire.jpg';
+// Introduce component state #13
```

Comment:
Please remove commented code :)

Reply:
changed as requested and pushed..thanks for your suggestion
.cheers